readme_renderer/clean.py

```diff
@@ -46,6 +86,7 @@
     "span": ["class"],
     "th": ["align"],
     "td": ["align"],
+    "div": ["align"],
```

Comment:
Can you revert the other changes to this file? This should be the only line changed.

Reply:
@di fixed